Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -1052,6 +1052,17 @@ public extension String {
         return String(dropLast(suffix.count))
     }
 
+    // https://www.hackingwithswift.com/articles/141/8-useful-swift-extensions
+    /// SwifterSwift: Adds prefix to the string.
+    ///
+    ///     "www.apple.com".withPrefix("https://") -> "https://www.apple.com"
+    ///
+    /// - Parameter prefix: Prefix to add to the string.
+    /// - Returns: The string with the prefix prepended.
+    func withPrefix(_ prefix: String) -> String {
+        if hasPrefix(prefix) { return self }
```

Comment:
```suggestion
        guard !hasPrefix(prefix) { return self }
```
One more little thing :))

Reply:
I like this suggestion. Using `guard` for an early exit looks nice and fits well with the other methods. Updated!